### PR TITLE
Fix emitter receiver nodes

### PR DESCRIPTION
**Pull request by angel-ayala**
Changed variables nomenclature for one more standardized and msg new formats.
Update related to #677

### CODE REVIEW

File: webots_ros2_msgs/srv/GetBool.srv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+bool value
```
Comment (omichel) [inline webots_ros2_msgs/srv/GetBool.srv:2]:
```suggestion
bool value

```

**Comment by omichel** [comment]
It doesn't compile (see the log of the ROS2 tests / industrial_ci)

**Review by ygoumaz — CHANGES_REQUESTED** [review]
This PR should only contain the fixes on Emitter and Receiver. The changes about the new Compass plugin should not be contained in this PR.

**Comment by ygoumaz** [comment]
Emitter and Receiver should still be updated as explained in https://github.com/cyberbotics/webots_ros2/pull/677#issuecomment-1482996502.

**Pull request closed without merging**